src/main/java/org/jboss/wise/shared/GWTClientConversationBean.java

```diff
@@ -288,6 +289,15 @@ private TreeElement wiseOutputTransfer(WiseTreeElement wte) {
             gTreeElement.setProtoType(pElement);
          }
 
+         Type[] typeArr = ((ParameterizedType)wte.getClassType()).getActualTypeArguments();
+         if (typeArr != null && typeArr.length > 0) {
+            String actualType = typeArr[0].toString();
+            gTreeElement.setRawType(actualType);
+         } else {
+            System.out.println("ERROR parameterizedType actualTypeArguments not found for "
```

Comment:
Lets create an issue for it and I'll make a checkin with all needed changes.

----- Original Message -----
> From: "Alessio Soldano" <notifications@github.com>
> To: "jboss-wise/wise-gwt-gui" <wise-gwt-gui@noreply.github.com>
> Cc: "rsearls" <rsearls@redhat.com>
> Sent: Thursday, May 21, 2015 10:54:25 AM
> Subject: Re: [wise-gwt-gui] Msg response collection (#3)
> 
> > @@ -288,6 +289,15 @@ private TreeElement wiseOutputTransfer(WiseTreeElement
> > wte) {
> >              gTreeElement.setProtoType(pElement);
> >           }
> >  
> > +         Type[] typeArr =
> > ((ParameterizedType)wte.getClassType()).getActualTypeArguments();
> > +         if (typeArr != null && typeArr.length > 0) {
> > +            String actualType = typeArr[0].toString();
> > +            gTreeElement.setRawType(actualType);
> > +         } else {
> > +            System.out.println("ERROR parameterizedType
> > actualTypeArguments not found for "
> 
> We should use proper logger instead of System.out (assuming this is not
> something meant to be returned as error to the user interface).
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/jboss-wise/wise-gwt-gui/pull/3/files#r30810061